index.js

```diff
@@ -9,7 +9,7 @@ const gitHubInjection = cb => {
     throw new TypeError('Callback is not a function');
   }
 
-  document.addEventListener('pjax:end', cb);
+  document.addEventListener('turbo:visit', cb);
```

Comment:
Both could be kept for consistency
```suggestion
  document.addEventListener('pjax:end', cb);
  document.addEventListener('turbo:visit', cb);
```